### DISCUSSION TITLE
Altair element

### DISCUSSION THREAD

**Discussion by dridk** [zauberzeug/nicegui#1909] [Ideas / Feature Requests]
Altair is a python library to generate vega.js plot .
https://altair-viz.github.io

It would be greate to have an element like plotly and matplotlib. 

```python
import Altair as alt 
import pandas as df 

df = ....
chart = alt.Chart(df).mark_lines()...
ui.altair(chart) 

```

**Comment by rodja** [comment]
Yes, I agree. Anyone interested in creating a pull request?

**Reply by dridk** [reply]
I can try. Do you have some documentation? I don't understand right now the meaning of your different mixins

**Reply by kk-deng** [reply]
I would love to use Altair with NiceGUI too. Any updates on this? Not sure if I could help too... @dridk

**Comment by falkoschindler** [comment]
There's already a feature request for Altair: #657
Let's continue the discussion over there. 🙂
